app/src/main/java/com/keylesspalace/tusky/MainActivity.kt

```diff
@@ -169,6 +171,8 @@ class MainActivity : BottomSheetActivity(), ActionButtonActivity, HasAndroidInje
         val hideTopToolbar = preferences.getBoolean(PrefKeys.HIDE_TOP_TOOLBAR, false)
         mainToolbar.visible(!hideTopToolbar)
 
+        mainToolbar.navigationIcon = FixedSizeDrawable(getDrawable(R.drawable.avatar_default), Utils.dpToPx(this, 36), Utils.dpToPx(this, 36))
```

Comment:
The navigation icon is not automatically sized correctly, and it ignores bounds set on the drawable, but it works when using this `FixedSizeDrawable` from Glide.

Reply:
What does 36 mean here?

Looks like magic number to me.

Reply:
I'm okay with it being a magic number but we use it 3 times so I would put it into constant.